ingestion/src/metadata/ingestion/source/metadata/atlas.py

```diff
@@ -49,10 +53,14 @@
 from metadata.ingestion.ometa.ometa_api import OpenMetadata
 from metadata.ingestion.source.database.column_type_parser import ColumnTypeParser
 from metadata.utils import fqn
+from metadata.utils.amundsen_helper import SERVICE_TYPE_MAPPER
+from metadata.utils.connections import get_connection
 from metadata.utils.logger import ingestion_logger
 
 logger = ingestion_logger()
 
+REQUEST_DESCRIPTION = "Atlas Metadata"
```

Comment:
I'm not sure if we should force this description

Reply:
Instead maybe we can apply tags

Reply:
Oh Ok 👍